modules/frontend/frontend.go

```diff
@@ -235,8 +235,8 @@ func newSpanMetricsMiddleware() Middleware {
 }
 
 // buildUpstreamRequestURI returns a uri based on the passed parameters
-// we do this because weaveworks/common uses the RequestURI field to translate from http.Request to httpgrpc.Request
-// https://github.com/weaveworks/common/blob/47e357f4e1badb7da17ad74bae63e228bdd76e8f/httpgrpc/server/server.go#L48
+// we do this because dskit/common uses the RequestURI field to translate from http.Request to httpgrpc.Request
+// https://github.com/dskit/common/blob/47e357f4e1badb7da17ad74bae63e228bdd76e8f/httpgrpc/server/server.go#L48
```

Comment:
I think this should be the correct line
```suggestion
// https://github.com/grafana/dskit/blob/740f56bd293423c5147773ce97264519f9fddc58/httpgrpc/server/server.go#L59
```